### PR TITLE
add interac icon

### PR DESCRIPTION
**Pull request by PatrickFang**
** Checklist **
- [x] All icons have a corresponding entry in `db/payment_icons.yml`
- [x] I have followed the icon guidelines detailed in the [CONTRIBUTING.md](https://github.com/activemerchant/payment_icons/blob/master/CONTRIBUTING.md) file
- [x] I have optimized the icon with [SVGO](https://jakearchibald.github.io/svgomg/)
- [x] I am confident that all icons are clear and easy to read/understand
- [ ] I have provided a link to the brand icon’s brand guidelines whenever possible.

If this pull request is not adding new icons, you can remove this checklist.

### CODE REVIEW

**Comment by NathanPJF** [comment]
You'll need to use an SVG that rounds these corners, @PatrickFang 



The template in the contributing guidelines is a good example.

File: app/assets/images/payment_icons/interac.svg [hunk 1]
```diff
@@ -0,0 +1 @@
...1.088l-.023.006c-.343.083-.725.169-.794.185a.674.674 0 0 1-.125.012" fill="#1A1919"/><path fill="#1A1919" d="M1.312 13.776V7.682l1.11-.263v6.094l-1.11.263"/><path d="M4.08 13.257a.686.686 0 0 0-1.372 0l.001 5.696c0 1.022.833 1.856 1.855 1.856l1.94-.002.001-2.377v-2.65c0-.3-.148-.579-.398-.746l-1.73-1.175v2.667a.148.148 0 0 1-.148.149.148.148 0 0 1-.146-.149l-.004-3.269" fill="#1A1919"/><path d="M5.005 9.008c-.717.165-.99.764-.99.764v-.428l-1 .237.002 2.735a1.019 1.019 0 0 1 1.053.184v-1.766c0-.423.211-.761.511-.827.227-.05.416.031.416.434v2.577l1.054-.248V9.945c0-.661-.254-1.123-1.046-.937" fill="#1A1919"/><path d="M20.315 5.107a.652.652 0 0 1 0-1.302.652.652 0 0 1 0 1.302m0-1.22a.57.57 0 0 0 0 1.139.57.57 0 0 0 0-1.14" fill="#1A1919"/><path d="M20.077 4.068h.598c.238 0 .43.077.43.283 0 .142-.098.252-.282.271v.002c.167.011.255.071.264.226.003.07.004.156.01.224.006.054.045.08.08.094h-.304c-.028-.02-.04-.053-.046-.097-.01-.066-.008-.128-.012-.208-.004-.12-.06-.173-.247-.173h-.22v.478h-.27v-1.1zm.486.49c.174 0 .27-.068.27-.179 0-.117-.076-.178-.255-.178h-.23v.357h.215z" fill="#1A1919"/></g><rect stroke-opacity=".07" stroke="#000" x=".5" y=".5" width="37" height="23" rx="2"/></g></svg>
```
Comment (tauthomas01) [inline app/assets/images/payment_icons/interac.svg]: There's an extra `id="pi-Interac-icon-sml"` that can be removed.

```suggestion
<svg role="img" aria-labelledby="pi-interac" width="38" height="24" viewBox="0 0 38 24" version="1.1" xmlns="http://www.w3.org/2000/svg"><title id="pi-interac">Interac</title><g stroke="none" stroke-width="1" fill="none" fill-rule="evenodd"><path fill="#FFF" d="M0 0h38v24H0z"/><g transform="translate(8 1)"><path d="M21.592 19.824c0 .977-.793 1.768-1.768 1.768H2.143c-.977 0-1.769-.791-1.769-1.768V2.142c0-.977.792-1.767 1.769-1.767h17.68c.976 0 1.769.79 1.769 1.767v17.682" fill="#E2A42E"/><path d="M19.824.374c.975 0 1.768.792 1.768 1.768v17.682c0 .977-.793 1.768-1.768 1.768H2.142c-.976 0-1.767-.791-1.767-1.768V2.142c0-.976.79-1.768 1.767-1.768h17.682m0-.374H2.142C.962 0 0 .96 0 2.142v17.682c0 1.18.961 2.142 2.142 2.142h17.682a2.145 2.145 0 0 0 2.142-2.142V2.142C21.966.961 21.006 0 19.824 0" fill="#FFFFFE"/><path d="M12.221 11.212l-.002-3.799.992-.235v.498s.257-.655.853-.807c.183-.046.276-.019.276-.019v.97s-.225-.01-.52.108c-.366.145-.55.46-.55.942v2.098l-1.049.244" fill="#1A1919"/><path d="M6.994 12.447s-.161-.241-.161-1.096V9.459l-.51.122V8.8l.51-.12v-.85l1.055-.248v.849l.745-.177v.778l-.745.177v1.926c0 .876.233 1.044.233 1.044l-1.127.267" fill="#1A1919"/><path d="M8.743 10.187c0-.675.096-1.167.303-1.546.245-.449.648-.74 1.236-.874 1.156-.263 1.573.421 1.557 1.339-.006.328-.004.488-.004.488l-2.038.48v.032c0 .642.135.946.541.86.351-.074.449-.294.476-.566.005-.044.007-.155.007-.155l.955-.23s.002.08 0 .166c-.008.365-.114 1.27-1.44 1.583-1.257.299-1.593-.47-1.593-1.577m1.58-1.713c-.337.077-.514.408-.522.957l1.026-.245c.002-.024.002-.08.002-.165-.002-.414-.13-.632-.505-.547" fill="#1A1919"/><path d="M17.908 7.966c-.039-1.157.26-2.076 1.551-2.368.824-.187 1.136.027 1.294.234.152.197.211.464.211.836l.001.07-1.01.24-.001-.141c0-.443-.122-.61-.443-.53-.382.096-.536.458-.536 1.179l.001.336c0 .732.1 1.068.54.976.382-.08.434-.415.44-.708l.003-.194 1.009-.238v.157c-.002.96-.508 1.566-1.457 1.786-1.306.307-1.562-.45-1.603-1.635" fill="#1A1919"/><path d="M14.417 9.585c0-.88.516-1.158 1.292-1.479.698-.288.712-.431.716-.683.003-.21-.094-.391-.442-.303-.319.081-.403.313-.42.497-.01.1-.007.196-.007.196l-.978.23s-.016-.301.067-.578c.157-.526.621-.88 1.422-1.065 1.04-.24 1.388.216 1.39.927V9.01c0 .814.15.933.15.933l-.96.227s-.09-.186-.13-.345c0 0-.21.53-.935.7-.761.18-1.165-.292-1.165-.94m2.001-1.201s-.2.147-.511.299c-.32.157-.465.35-.465.647 0 .257.16.426.45.354.31-.078.526-.368.526-.77v-.53" fill="#1A1919"/><path d="M1.48 15.11a.61.61 0 0 1-.51-.942l.007-.008.011-.003 1.435-.338v1.089l-.023.005c-.325.078-.714.167-.794.184a.551.551 0 0 1-.125.013" fill="#1A1919"/><path d="M1.48 16.504a.609.609 0 0 1-.51-.94l.007-.01.011-.003 1.435-.338v1.089l-.023.006a37.76 37.76 0 0 1-.794.184.647.647 0 0 1-.125.012" fill="#1A1919"/><path d="M1.48 17.9a.61.61 0 0 1-.51-.94l.007-.011 1.446-.34v1.088l-.023.006c-.343.083-.725.169-.794.185a.674.674 0 0 1-.125.012" fill="#1A1919"/><path fill="#1A1919" d="M1.312 13.776V7.682l1.11-.263v6.094l-1.11.263"/><path d="M4.08 13.257a.686.686 0 0 0-1.372 0l.001 5.696c0 1.022.833 1.856 1.855 1.856l1.94-.002.001-2.377v-2.65c0-.3-.148-.579-.398-.746l-1.73-1.175v2.667a.148.148 0 0 1-.148.149.148.148 0 0 1-.146-.149l-.004-3.269" fill="#1A1919"/><path d="M5.005 9.008c-.717.165-.99.764-.99.764v-.428l-1 .237.002 2.735a1.019 1.019 0 0 1 1.053.184v-1.766c0-.423.211-.761.511-.827.227-.05.416.031.416.434v2.577l1.054-.248V9.945c0-.661-.254-1.123-1.046-.937" fill="#1A1919"/><path d="M20.315 5.107a.652.652 0 0 1 0-1.302.652.652 0 0 1 0 1.302m0-1.22a.57.57 0 0 0 0 1.139.57.57 0 0 0 0-1.14" fill="#1A1919"/><path d="M20.077 4.068h.598c.238 0 .43.077.43.283 0 .142-.098.252-.282.271v.002c.167.011.255.071.264.226.003.07.004.156.01.224.006.054.045.08.08.094h-.304c-.028-.02-.04-.053-.046-.097-.01-.066-.008-.128-.012-.208-.004-.12-.06-.173-.247-.173h-.22v.478h-.27v-1.1zm.486.49c.174 0 .27-.068.27-.179 0-.117-.076-.178-.255-.178h-.23v.357h.215z" fill="#1A1919"/></g><rect stroke-opacity=".07" stroke="#000" x=".5" y=".5" width="37" height="23" rx="2"/></g></svg>
```

**Comment by tauthomas01** [comment]
Hi @PatrickFang ,

Is your PR ready to review? I see a new icon added in this PR. Can you change the title of the PR or create a separate PR for the new icon? It will need some cleaning as well.

Thank you

**Comment by PatrickFang** [comment]
yep i will update the icon appropriately, still working it through. thanks!

**Comment by Guillaumegranger1** [comment]
There was some extra layers I could remove without changing the icon appearance. Here's a [screen recording](https://screenshot.click/screencast_2019-05-22_11-12-01.mp4) to show what I mean. And here's an SVG export I made from that cleaned up file if that helps:

[interac-cleanedup.svg.zip](https://github.com/activemerchant/payment_icons/files/3208038/interac-cleanedup.svg.zip)

**Comment by PatrickFang** [comment]
@Guillaumegranger1 thanks for the asset! updated to use the new asset you provided, please review again!

**Comment by tauthomas01** [comment]
@PatrickFang Could you run a pass over SVGO as well once Guillaume approves? Thank you!

**Pull request merged**